extra-cpp-bindings/src/walletconnect.rs

```diff
@@ -151,56 +151,56 @@ async fn do_walletconnect_set_callback(
     client.run_callback(Box::new(move |message| {
         match message.state {
             ClientChannelMessageType::Connected => {
-                println!("Connected");
```

Comment:
maybe those printouts can be removed or kept? I think that `do_walletconnect_set_callback` is more for example command-line callback setting; in the unreal plugin, it's using its own callback?

Reply:
Yes, printouts in the binding should be removed in my opinion. Users would be very confused because they don't know where those printouts come from. 

Right, unreal or cpp example use its own callback, no need to printout in the binding, considering all of them are internal debug logs.

Let me tidy them up.